src/routes/measurement-threshold/find.ts

```diff
@@ -66,8 +66,7 @@ const controllerGeneratorOptions: ControllerGeneratorOptionsWithClientAndSupplie
     pinHashId: Joi.string().allow(null).default(null),
     type: Joi.string().valid('quantity', 'port'),
   }),
-  // rights to be updated to 'Threshold' which will be added soon
-  right: { environment: 'ISSUES', supplier: 'READ' },
+  right: { environment: 'THRESHOLDS', supplier: 'READ' },
```

Comment:
In the context of the general-information-markdown this makes no sense I think. What does it mean that for this route you need a read right from the supplier environment if we only know the ENVIRONMENT_ADMIN right for connectivity environments. If we need to access this from the connectivity environment this needs to be 
```supplier: 'ENVIRONMENT_ADMIN'```
However, at this point I think it is most likely that you don't need the supplier value here at all.

Reply:
Yes, thanks for this